internal/cli/iam/projects/create.go

```diff
@@ -74,51 +76,90 @@ func (opts *CreateOpts) newRegionUsageRestrictions() string {
 }
 
 func (opts *CreateOpts) newCreateProjectOptions() *atlas.CreateProjectOptions {
-	return &atlas.CreateProjectOptions{ProjectOwnerID: opts.projectOwnerID}
+	return &atlas.CreateProjectOptions{
+		ProjectOwnerID: opts.projectOwnerID}
 }
 
 func (opts *CreateOpts) validateOwnerID() error {
 	if opts.projectOwnerID == "" {
 		return nil
 	}
 
-	if config.Service() != config.OpsManagerService {
+	sv, constrain, err := opts.newOpsManagerConstrain()
+	if err != nil {
+		return err
+	}
+
+	if sv == nil || constrain == nil {
 		return nil
 	}
 
-	v, err := opts.store.ServiceVersion()
-	if err != nil {
-		return err
+	if !constrain.Check(sv) {
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.OwnerID)
 	}
 
-	sv, err := cli.ParseServiceVersion(v)
+	return nil
+}
+
+func (opts *CreateOpts) validateWithoutDefaultAlertSettings() error {
+	if !opts.withoutDefaultAlertSettings {
+		return nil
+	}
+
+	sv, constrain, err := opts.newOpsManagerConstrain()
 	if err != nil {
 		return err
 	}
 
-	constrain, _ := semver.NewConstraint(">= 6.0")
+	if sv == nil || constrain == nil {
+		return nil
+	}
 
 	if !constrain.Check(sv) {
-		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.OwnerID)
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.WithoutDefaultAlertSettings)
 	}
 
 	return nil
 }
 
-// mongocli iam project(s) create <name> [--orgId orgId] [--ownerID ownerID].
+func (opts *CreateOpts) newOpsManagerConstrain() (*semver.Version, *semver.Constraints, error) {
+	if config.Service() != config.OpsManagerService {
+		return nil, nil, nil
+	}
+
+	v, err := opts.store.ServiceVersion()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	sv, err := cli.ParseServiceVersion(v)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	constrain, err := semver.NewConstraint(">= 6.0")
+
+	return sv, constrain, err
+}
+
+// mongocli iam project(s) create <name> [--orgId orgId] [--ownerID ownerID] [--withoutDefaultAlertSettings].
 func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{}
 	opts.Template = atlasCreateTemplate
 	cmd := &cobra.Command{
 		Use:   "create <name>",
```

Comment:
thanks for annotating the params, let's make them match 
```suggestion
		Use:   "create <projectName>",
```